modules/workbench/modules/reporting/assets/schemas/user.json

```diff
@@ -224,5 +224,65 @@
     "type": "STRING",
     "description": "If the institutional_role_enum is \"other\", custom description\nof this user's role in the institution.",
     "mode": "NULLABLE"
+  },
+  {
+    "name": "degrees",
+    "type": "STRING",
+    "description": "List of user's degrees.",
+    "mode": "NULLABLE"
+  },
+  {
+    "name": "race",
```

Comment:
Per other ticket I would pluralize these column names, if they can be repeated. I would also specify in the description that: "comma-delimited if multiple answers were provided"

Reply:
done